docs/modules/parameters/pages/action-scope.adoc

```diff
@@ -0,0 +1,82 @@
+= Action scope
+
+== Description
+
+Defines the scope of an action
```

Comment:
Why action scope, but not for example type?

Reply:
refactored